### ISSUE TITLE
early mitosis 

### ISSUE THREAD

**Issue by Antonialock** [pombase/fypo#2562]
**Q1: are these the same thing?**

premature mitotic G2/M transition 
A cell cycle phenotype in which the G2/M transition of the mitotic cell cycle begins earlier than normal. The duration of G2 phase is thus shorter than normal.

premature mitosis 
A cellular process phenotype in which cells enter mitosis earlier, relative to cell size or DNA integrity checkpoint status, than normal.

**Q2. Should one or both lead to**
mitotic catastrophe
An inviable phenotype in which a cell enters mitosis prematurely, undergoes abnormal mitotic chromosome segregation, and subsequently dies.

**Comment by ValWood** [comment]
the way mitosis is defined it  refers to the whole of M-phase (includes chromosome condensation), so in this case I think these two terms are describing the same thing... an early G2/M transition

They would not necessarily lead to mitotic catastrophe though, because cells are often viable and small.

I think 'mitotic catastrophe should be tweaked. It is really entering 'anaphase' early rather than mitosis.
https://en.wikipedia.org/wiki/Mitotic_catastrophe

**Comment by JackyVH** [comment]
Hi Val and Antonia
I'm at the retreat so this is short but the two descriptions sound different to me, in the first the the entry in to M is early relative to size but (presumably) not the DNA integrity checkpoint so cells  will be small but viable where as in the second one cells enter M early compared to both size and integrity checkpoint so would  be a mitotic catastrophe.

Does this make sense

Jacky

**Comment by ValWood** [comment]
premature mitosis 
says
A cellular process phenotype in which cells enter mitosis earlier, relative to cell size or DNA integrity checkpoint status, than normal.

so currently its referring to size OR checkpoint status rather than AND
so I guess only one of these could be occurring. Wouldn't this make them the same?

There are only currently 4 annotations to each of these terms so we could have a look at them all and see how consistent they are.
Will wait for Midori too, and follow up with you before we change anything.

Enjoy the retreat.

**Comment by Antonialock** [comment]
I also used premature mitotic G2/M transition 
in PMID:24728197 https://curation.pombase.org/pombe/curs/7b0762e353cab896

Also, in this one a cdc25-22, atf-OP mutant enter mitosis early and ends up cut. I annotated it to
mitotic catastrophe with cut 
I don't know if the DNA integrity checkpoint is affected, just the G2/M happens 'earlier' and that you end up with cut cells. Is that right?

**Comment by ValWood** [comment]
I'm a bit confused with these.

Re
Also, in this one a cdc25-22, atf-OP mutant enter mitosis early and ends up cut. I annotated it to
mitotic catastrophe with cut 
I don't know if the DNA integrity checkpoint is affected, just the G2/M happens 'earlier' and that you end up with cut cells. Is that right?

How do you know G/2 M is happening earlier here?. G2/M and metaphase/anaphase are  separate.
G2/M is when the cells enter M-phase (at prophase). Chromosome segregation is metaphase/anaphase.

Jacky also says
"I see what you mean about premature mitosis in both situation cells enter mitosis early.  However they do have different outcomes but I suppose this is reflected in child terms"

I think it might be useful to have a discussion about these  different phenotypes and what they mean when Jacky has time.

I think sometimes they are one thing, and sometimes they are 2 things simultaneously.

**Comment by Antonialock** [comment]
I "know" it because the authors explicitly state it.

I don't know if they are wrong, did you look at the paper?

**Comment by ValWood** [comment]
Isn't cdc25-22 elongated? that's doing G2/M transition late not early.

I think we need to review these terms and separate what a biologist actually observes from what they say
'premature mitosis' in the text can mean a number of things, I think.

This is also related to a comment Jacky made previously
"As an aside and take no notice of it, I always have a bit of a problem with this, it seems to me that for the small cells mitosis is not really premature, they have the same cycle time as wild type cells but the cell cycle phases are different G1 is longer and G2 is shorter."

So we may might need to be more explicit what the terms are to be used for.

As a start, Antonia could you pop an image of  the phenotype or the fig  you are curating?

**Comment by ValWood** [comment]
atf1 doesn't regulate the G2/M transition. It might regulate genes *required* for G2/M but the transition is regulated post-translationally. I don't see experiments for this?

They say:
Very little is known about the transcription factors that drive G2/M progression. Our lab was the first to identify that the tran- scription factor Atf1 controls the expression of the mitotic cyclin cdc13+, related to G2/M transition in S. pombe 

so all they are saying is that Atf1 controls the expression of cdc13, and cdc13 is related to G2/M.

**Comment by ValWood** [comment]
I can't find cdc25-22 in this paper?

**Comment by Antonialock** [comment]
they delete atf1 they get less cdc13 and mitosis is delayed, 
if they overexpress atf1 they get more cdc13 and mitosis is advanced

sems like the transition is at least in part regulated by the expression levels of cdc13

**Comment by Antonialock** [comment]
err, most of their genotypes are double mutants containing cdc25...

**Comment by ValWood** [comment]
cdc25-22-  thats just for synchronization (cdc25-22 block and release), so thats 'background'

So cyclin levels are important for G2/M transition, and atf1 has some role in their expression... you can do phenotypes for these.

I don't think it should be GO terms here.  Atf1 isn't really regulating when the transition occurs , that part happens later (and it requires the cyclin to be present, because all of the pathways converge on the CDk-cyclin). But yes if you make lots more, or lots less cyclin than you would have in a WT you will mess up the timing.

**Comment by ValWood** [comment]
i.e it's indirect.
"Hence through Plo1, Atf1 can indirectly influence the transcriptional requirements of M–G1 transition, and through Cdc10 (MBF subunit) and Yox1 (MBF regulator) it does the same for G1/S transition."

In S. cerevisiae its a bit different(I think)  and there is a direct role for transcription in regulation at G1/S to increase the cyclin level via a positive feedback loop. In pombe I don't think this is the case....

**Comment by ValWood** [comment]
I'm looking at the other paper. No wonder I couldn't see it.

**Comment by Antonialock** [comment]
"But yes if you make lots more, or lots less cyclin than you would have in a WT you will mess up the timing." yes I agree with that this is not evidence of regulation 

Hmmmmmmmm, it is really unclear, I agree I wouldn't annotate RNA pol II to "G2/M transition" just because the expression of cdc13 depends on it.

However in this paper they show that atf1 is the TF for cdc13 so in a way it is involved in the transition. Presumably an event (some stress?) could inhibit atf1, thereby delaying G2/M?

**Comment by ValWood** [comment]
Hang on....
In cdc25-22  at restrictive temperature cells are elongated

overexpress atf1 and they are normal length after temperature shift
delete atf1 and they are still elongated after temperature shift

but then, 
in cdc25-22 atf1 OEXP  cells are elongated and cut at 37

that looks contradictory?
Maybe I'm missing something?

**Comment by mah11** [comment]
I don't think I can help (at least not off the top of my head) with annotating atf1 or whatever other genes; I haven't looked at the paper. Just some thoughts on the ontologies ..

Defining mitotic catastrophe as premature anaphase + abnormal chromosome separation would be straightforward enough. The current definition is just something I got by polishing some ideas from Val, so it's readily adjustable.

Correction to one of Val's comments above: "Chromosome segregation is metaphase/anaphase" should say "chromosome *separation* is metaphase/anaphase".

Now on to the head-melty part.

In FYPO, premature mitotic G2/M transition and premature mitosis point to different GO terms. Mitosis and the G2/M transition aren't quite identical; if they were, GO wouldn't need separate phase and process terms.

GO defines the G2/M transition in terms of cyclin/CDK levels, and as the cell committing to M phase. Mitosis is defined as the set of steps that add up to nuclear division.

For phenotypes, I think we have two options:

We could merge the terms, on the grounds that it's usually going to be difficult, if not impossible, to look at experimental results and make the distinction that accounts for the separate GO terms. If we merge, we'll have to decide which name and GO xp to use (I have a slight preference for "mitosis" because that's what most of the experiments show).

OR

We could keep the terms separate, and only use the premature G2/M transition term for experiments that actually look at cyclin/CDK levels. That would match GO most closely, and be very precise, but it's probably going to be a pain to make the distinction clear enough for community curators to understand and use the terms as we intend.

**Comment by ValWood** [comment]
- Presumably an event (some stress?) could inhibit atf1, thereby delaying G2/M?

Yes, thats true but it hasn't been demonstrated.

**Comment by mah11** [comment]
I'm aware that I haven't fully covered the "small cell" angle here. It's sort of related to Val's other ticket about "wee" (#2544).

I'm happy to revisit how we define "premature mitosis" too, whether or not it gets merged with the transition term. I have a vague recollection that that's another definition that we thought might only be "close enough for now" ages ago, not fully perfected.

**Comment by mah11** [comment]
And I'm still staying away from the annotation side of this ticket, because on that side I'm well and truly confused!

**Comment by ValWood** [comment]
I like Midori's second option. I think it would be more scientifically useful to make this distinction. I'm happy to go through the existing annotations because there are not that many.
Lets table this for a curator meeting. 
In the meantime Antonia I would put the phenotypes in the session, which match most closely but leave out the GO terms.
I will compile a list of existing usage for the next curator meeting. I think this is worth looking at in detail.

Phew!

**Comment by Antonialock** [comment]
> overexpress atf1 and they are normal length after temperature shift
> delete atf1 and they are still elongated after temperature shift
> 
> but then in cdc25-22 atf1 OEXP cells are elongated and cut at 37

Yes I agree that fig1c looks confusing. I assumed that their imageJ analysis was rubbish and that the cells are actually bigger (compare fig 1b top left to the one diagonally down on the right)...If they hadn't have had the last few figures in the paper (showing atf1 binding the cdc13 promoter) I wouldn't have bothered with GO terms.

**Comment by Antonialock** [comment]
> but leave out the GO terms.

You mean the Biological Process term right? Not the MF or CC?

**Comment by ValWood** [comment]
totally fine to have
transcriptional activator activity, RNA polymerase II core promoter proximal region sequence-specific binding with cdc13 as substrate.

**Comment by ValWood** [comment]
You mean the Biological Process term right? Not the MF or CC?
Yep!

**Comment by Antonialock** [comment]
yeah ok, I see what you mean....soo hard to keep the distinctions, but I guess that's what annotation reviews are for!

**Comment by mah11** [comment]
OK, I'll leave "premature mitosis" and "premature G2/M transition" in place for now. Might add comments to help explain the distinction if I get inspired.

Do you still want to adjust the "mitotic catastrophe" definition? History in #190 & #203.

**Comment by mah11** [comment]
comments added

I think the only ontology (as opposed to annotation) thing outstanding is whether to change the "mitotic catastrophe" definition.

**Comment by mah11** [comment]
comments in
edit file: deaef0426ccef63b4fdb8e7b0497838810101852
release: b0b60d0941f0bc92c4971f6a2dad85df58ea1031

**Comment by ValWood** [comment]
I'm a bit lost with this one.


Antonia can you summarize the outstanding questions?

If you used 
FYPO:0001046 - premature mitosis
for an annotation:
atf1, cdc25, mcm4, orc1, stm1
can you confirm what the phenotype was (paper/figure)? then we can figure out if they are  
"premature G2/M" (inferred when vegetative  cells are viable, but divide at small size), or something else....

**Comment by Antonialock** [comment]
Outstanding: 
-what do we have annotated to premature mitosis at the moment
-do we keep both "premature mitosis" and "premature G2/M transition" or merge them? Or something else (see below)?

Defs
**Premature mitosis** - A cellular process phenotype in which cells enter mitosis earlier, relative to cell size or DNA integrity checkpoint status, than normal.
**Premature G2/M transition** -  	A cell cycle phenotype in which the G2/M transition of the mitotic cell cycle begins earlier than normal. The duration of G2 phase is thus shorter than normal. The mitotic G2/M transition is the point at which a cell commits to entering M phase, and begins when the kinase activity of M cyclin/CDK complex reaches a threshold high enough for the cell cycle to proceed.

From Midori

> GO defines the G2/M transition in terms of cyclin/CDK levels, and as the cell committing to M phase. Mitosis is defined as the set of steps that add up to nuclear division.

Midori reckoned that we could either merge them (easiest option) or keep them separate, only using the premature G2/M transition term for experiments that look at cyclin/CDK levels (will be a pain to enforce). 

we had 5 genes annotated to premature mitosis. I removed the annotation for 3 of them. The two remaining ones really seemed "premature" as in they do not begin/complete S phase. Consequently mitosis goes wrong. 

Orc1 was aalready annotated to "mitosis with unreplicated DNA" (child term of premature mitosis), I moved the other gene down to this too.

mitosis with unreplicated DNA - A cellular process phenotype in which cells attempt to undergo mitosis before they have **begun** to replicate genomic DNA.

This could be broadened to  "A cellular process phenotype in which cells attempt to undergo mitosis before they have **completed** to replicate genomic DNA." and merged with premature mitosis? And then we can keep the premature G2/M transition for anything where the cells actually have transitioned to G2?

**mcm4** - these guys really seem like the are doing mitosis prematurely, they are doing it without completing Sphase
"Flow cytometry showed that mcm4-deficient spores did not undergo S phase, but remained with a 1C DNA content. Even though Sphase had not takenplace, 15-20% of cells attempted to carry out mitosis with a similar timing to the wild-type culture (14 h). Cells which had attempted division showed an aberrant mitotic appearance and formed mitotic spindles (Figure 5B-K), indicating that they were not undergoing unscheduled septation."
I changed this annotation to "mitosis with unreplicated DNA"

**orc1** - annotated to the child term "mitosis with unreplicated DNA"
orp1-4 cells do not begin DNA replication (most of them), but they still try to divide



Deleted ones:
**stm1** - I decided to delete this annotation. The experiments were done in diploid cells. Overexpression of stm1 lead to increased haploidization. They say this is due to premature mitosis (mitosis before genome finished replicating) but I don't think they have evidence for that, it could be due to other things
**atf1** - community curated annotation with microscopy evidence (short paper only showing gene expression data). They don't mention premature mitosis in the paper, only that atf1 somehow regulates cell cycle genes (and the overexpression mutant is small). I deleted the annotation.
**cdc25** - in cdc25-OP the cell size is smaller but the number of binucleate cells is similar. The authors therefore believe that the reduction in cell size must be due to a shorter G2 period and an advancement of mitosis. I removed the premature mitosis annotation - it is a speculation/conclusion not an observation.

**Comment by ValWood** [comment]
OK lets check what we have outstanding after you have removed the incorrect ones.



The only thing I am sure about, I don't want to change the definition of G2/M transition.
"GO defines the G2/M transition in terms of cyclin/CDK levels, and as the cell committing to M phase."
This is absolutely what the G2/M transition is.

 - G2/M transition term for experiments that look at cyclin/CDK levels (will be a pain to enforce).

I don't think  the cyclin/CDK levels need to be  levels need to be  looked at. We know that if cells are elongated  that cdc2 is activated late, and small cells vice versa. This phenotype  is used as a proxy for active Cdk/cyclin levels.
We know that all vegetative elongated phenotype have a block  or delay in G2/M transition.
We know through biological knowledge that this  phenotype is indicative of abnormal  cyclin/CDC activity.

In the atf1 and cdc25 cases above premature mitosis is another way of saying "premature G2/M transition" It follows that everything which has premature G2/M transition  has_output premature mitosis (if they are different).

So in summary, I am happy for them to be merged, but the definition of the G2/M term needs to stay the same.....

I don't think they are the same though ...mitosis is broader and the delay could be anywhere in the process I presume?  ...

How many things are left annotated to "premature mitosis" directly?

**Comment by Antonialock** [comment]
none: two genes (mcm4 and orc1) are annotated to the child term of premature mitosis "mitosis with unreplicated DNA" 

for orc1 they actually do check that the cells haven't started replicating their DNA. I'm not 100% sure about mcm4 so I wonder if we should relax the def to say
mitosis with unreplicated DNA - A cellular process phenotype in which cells attempt to undergo mitosis before they have **completed** genomic DNA replication.


(instead of:
mitosis with unreplicated DNA - A cellular process phenotype in which cells attempt to undergo mitosis **before** they have begun to replicate genomic DNA. )

**Comment by ValWood** [comment]
So we could
i) make a output_of  link between  premature G2/M transition and premature mitosis 
(would that make sense?)

ii) make  "premature mitosis " not for direct annotation. We can change this if we find an example where we clearly need to use it?

**Comment by Antonialock** [comment]
@mah11

**Comment by mah11** [comment]
> I wonder if we should relax the def to say
> mitosis with unreplicated DNA - A cellular process phenotype in which cells attempt to undergo 
> mitosis before they have **completed** genomic DNA replication.

Sorry, we cannot do this. In the replication field, "unreplicated" really means that no (detectable) replication has taken place. If replication has begun but not completed, DNA is said to be "partially replicated".

**Comment by mah11** [comment]
>  make a output_of link between premature G2/M transition and premature mitosis
> (would that make sense?)

What you said above is more consistent with has_output than output_of (premature G2/M transition has_output premature mitosis). That would say that any time a cell goes through the G2/M transition prematurely, it attempts mitosis too early too. I think we do want to say that.

It leaves open the possibility that some cell could undergo mitosis early for some other reason (attempting nuclear division in spite of otherwise not having undergone the G2/M transition as determined by cyclin/CBK levels). I don't actually know whether that's even been observed in real-life cells; the has_output link merely doesn't rule it out.

Conversely, if we used premature mitosis output_of premature G2/M transition, that would allow for a situation in which a cell could do the premature G2/M transition early but still not undergo mitosis too early. I think we don't want to imply that?

> make "premature mitosis " not for direct annotation. We can change this if we find an example where we clearly need to use it?

no objection, but might be best done in the soft-check file rather than via the not-for-annotation subset baked into the ontology

and just an aside, without consequence for the ontology:

> We know that if cells are elongated that cdc2 is activated late, and small cells vice versa. This phenotype is used as a proxy for active Cdk/cyclin levels.
> We know that all vegetative elongated phenotype have a block or delay in G2/M transition.
> We know through biological knowledge that this phenotype is indicative of abnormal cyclin/CDC activity.

This is biological knowledge that I didn't have before I read the comments on this ticket from yesterday (and of course it's specific to fission yeasts).

**Comment by ValWood** [comment]
correction
> We know through biological knowledge that this phenotype is indicative of abnormal cyclin/CDC activity.

I should probably say CDK regulation (although that affects CDK activity indirectly)

Yep, if a vegetative cell is "classical" elongated  (and not multiseptate, or diploid, or germinating, which are sometimes elongated for other reasons)  it is  doing G2/M later than usual.

As  G2/M has to occur through cdc2 that places these phenotypes upstream of cdc2 somewhere...

so either in this pathway:
https://cloud.githubusercontent.com/assets/7359272/18444623/fa5df2ce-7911-11e6-98a7-d75c2e841641.jpg

or indirectly upstream
(which is around ~800 gene products spanning transcription, translation, splicing, etc)

This is why causally upstream isn't so useful.... we could eventually end up with everything annotated to everything if we annotate all upstream causal effects....

I'm having issues convincing people with some of the matrix outliers though.....will be asking your opinions soon..... once I have collated.....its interesting (to me ;)

**Comment by ValWood** [comment]
Also,
What I want to avoid is users in canto seeing

abnormal G2/M transition
--abnormal mitosis

it would be OK if they see
abnormal mitosis
--abnormal G2/M transition

I can *never* remember the way these 2 relationships work in the Canto display.
Maybe this time  it will sink in......

**Comment by mah11** [comment]
We shouldn't choose ontology relations based on Canto's behavior; that's backasswards. Instead, we should put the most logically and biologically accurate relation in the ontology, and then make sure Canto handles it as we want.

For G2/M transition and mitosis, has_output looks more accurate as noted in my last comment.

On to the Canto tangent! Wheeee!

According to the wiki (https://curation.pombase.org/pombase-trac/wiki/RelationsAndTransitivity), for has_output we expect to start with the output (e.g. premature mitosis) and be offered the thing that has the output (e.g. premature G2/M transition). I've just checked with some terms that are already linked by has_output, and Canto is doing what we want there.

The wiki says that as of the last time we thought about it - ages ago - we hadn't properly worked out what we expect for output_of. What it's doing right now is offer the output if you search for the thing on the other side of the link. For example, FYPO has the link  'abnormal Golgi morphology' output_of 'abnormal Golgi organization'. For 'abnormal Golgi organization' Canto offers 'abnormal Golgi morphology', whereas if I start with 'abnormal Golgi morphology' it doesn't offer 'abnormal Golgi organization'.

Based of what you've just said here, I now suspect that it would be better the other way round. If you agree, I'll open a Canto ticket.

**Comment by ValWood** [comment]
I think so, but I can't tell based on the output_of examples above.
Do we have any other examples to check?

**Comment by mah11** [comment]
well, there are almost 80 output_of links in FYPO ....

**Comment by ValWood** [comment]
OK this example makes more sense for me:

FYPO: A FYPO:0002458 viable branched vegetative cell output_of B FYPO:0001972 abnormal cytokinetic cell separation resulting in septated cell
Meaning: If you see a viable branched vegetative cell, you can infer that abnormal cytokinetic cell separation resulting in septated cell took place.
Search results: A query for FYPO:0001972 should retrieve genes annotated to FYPO:0002458 as well as genes annotated directly to FYPO:0001972.


here is it 
viable branched vegetative cell output_of B FYPO:0001972 abnormal cytokinetic cell separation resulting in septated cell

and viable branched vegetative cell 
would appear as a decendant if you search on 
 abnormal cytokinetic cell separation resulting in septated cell

...if so, I agree

**Comment by mah11** [comment]
hokay, https://github.com/pombase/canto/issues/1307

**Comment by mah11** [comment]
is there anything to do for this one?

**Comment by ValWood** [comment]
Lets say no
